app/components/ConnectorItem.jsx

```diff
@@ -5,13 +5,24 @@ import { translate } from '../plugins/preact-polyglot'
 
 const ConnectorItem = ({ title, subtitle, connected, slug, iconName, backgroundCSS = 'white', enableDefaultIcon = false, router }) => (
   <Link class='item-wrapper' to={`${router.location.pathname}/${slug}`}>
+    {backgroundCSS &&
```

Comment:
I may be missing some context here, but if you give `backgroundCSS` a default value (white) then `!backgroundCSS`will never happen, isn't it ?